polaris.shopify.com/src/components/ComponentExamples/ComponentExamples.module.scss

```diff
@@ -26,11 +26,20 @@
     }
 
     &[aria-selected='true'] {
-      background: var(--search-highlight-color);
-      &:focus {
-        box-shadow: var(--focus-outline);
-      }
+      background: var(--surface-active);
```

Comment:
these styles were broken/wrong so just updating them now to work on focus and selected states